### DISCUSSION TITLE
Growatt Battery System implemenet

### DISCUSSION THREAD

**Discussion by cracyfloyd** [johanmeijer/grott#273] [Ideas]
I use a Growatt Min2500XH with an XH Battery System. Its possible to implement the battery values ?

**Comment by johanmeijer** [comment]
Yes that is possible. I have not defined a Layout for the Min type inverters (with battery). 
I will plan to create one. I think the solar part should be shown by default (can you confirm?).

**Comment by cracyfloyd** [comment]
Nice to hear. It would be great.
yes i can confirm the solar part works with the defaults.
thnx a lot.
